### ISSUE TITLE
Power Gen Sidemission stuck

### ISSUE THREAD

**Issue by CurePindal** [DUWS-R-Team/DUWS-R#168]
Got a sidemission to sabotage a power generator. Had no clue what to do with the generator, so I blew it up with a explosive pack. Nothing happened and since then the mission is stuck.
I assume it was not the job to blow it up? But if so there seems to be a mission failure condition missing?!

**Comment by CurePindal** [comment]
Yep, checked the code. Youre suposed to use the "sabotage" option from the action menu. And the script has no failure condition in case the generator is destroyed.

**Comment by ultranoobian** [comment]
I made the Transformer invincible instead, since the mission is to sabotage, not destroy the transformer.
Its currently released under [v0.6.1x001](https://github.com/ultranoobian/DUWS-R/tree/v0.6.1x001)
